boards/adi/max32672evkit/doc/index.rst

```diff
@@ -0,0 +1,368 @@
+.. _max32672_evkit:
+
+MAX32672EVKIT
+#############
+
+Overview
+********
+The MAX32672 evaluation kit (EV kit) provides a platform
+for evaluating the capabilities of the MAX32672 microcontroller,
+which is a small, high-reliability, ultra-low power,
+32-bit microcontroller. The MAX32672 is a secure and
+cost-effective solution for motion/motor control, industrial
+sensors, and battery-powered medical devices and offers
+legacy designs an easy, cost-optimal upgrade path from
+8-bit or 16-bit microcontrollers.
```

Comment:
You are wrapping very aggressively, which hurt readability and will generate larger than necessary diffs in the future.
Please adjust line length to 100 characters in your editor and update RST files in this PR accordingly :)

Reply:
Updated thanks.